### PR TITLE
Auto assign action

### PR DESCRIPTION
**Pull request by eranelbaz**
auto-assign the pr creator as the `assignee` for the pr

### CODE REVIEW

File: .github/workflows/pr-auto-assign.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: 'Auto Assign PR to author'
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  add-assignee:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
```
Comment (roni-frantchi) [inline .github/workflows/pr-auto-assign.yml:9]: IMO this kind of action should be quick or fail fast

```suggestion
    timeout-minutes: 1
```

File: .github/workflows/pr-auto-assign.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    steps:
+      - uses: kentaro-m/auto-assign-action@v1.2.1
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (roni-frantchi) [inline .github/workflows/pr-auto-assign.yml:13]:
```suggestion
          repo-token: "${{ secrets.GITHUB_TOKEN }}"
          
```

File: .github/auto_assign.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# used by ./workflows/pr-auto-assign.yml
+addAssignees: author
+runOnDraft: true
```
Comment (roni-frantchi) [inline .github/auto_assign.yml:3]:
```suggestion
runOnDraft: true

```

**Review by roni-frantchi — APPROVED** [review]
What would happen with Renovate PRs? 🤔

**Comment by eranelbaz** [comment]
> What would happen with Renovate PRs? 🤔

We didn't configure renovate for this repo

**Pull request merged**